keps/sig-storage/3476-volume-group-snapshot/README.md

```diff
@@ -159,9 +159,13 @@ Note: In the following, we will use VolumeGroupSnapshot Controller to refer to t
 * The controller will retrieve all volumeSnapshotHandles in the Volume Group Snapshot from the CSI CreateVolumeGroupSnapshotResponse, create VolumeSnapshotContents pointing to the volumeSnapshotHandles. Then the controller will create VolumeSnapshots pointing to the VolumeSnapshotContents.
 * CreateVolumeGroupSnapshot CSI function response
   * The CreateVolumeGroupSnapshot CSI function should return a list of snapshots (Snapshot message defined in CSI Spec) in its response. The VolumeGroupSnapshot controller can use the returned list of snapshots to construct corresponding individual VolumeSnapshotContents and VolumeSnapshots, wait for VolumeSnapshots and VolumeSnapshotContents to be bound, and update SnapshotList in the VolumeGroupSnapshot Status and SnapshotContentList in the VolumeGroupSnapshotContent Status.
+ * Individual VolumeSnapshots will be named in this format:
+   * snap+<the first 4 letters of VolumeGroupSnapshot name>+<PVC name>
+     (truncate if it exceeds the max length)
+   * If the exact same name already exists, append a "1" at the end. If that still exists, replace the suffix "1" with "2", and so on.
```

Comment:
I think this could still be confusing to users. What if we use some hashing logic similar to PV names and then add labels to make it easy to search.

Reply:
Sure

Reply:
I thought PV name is (PVC)-(PVC UUID)?

Reply:
Yes looks like it's "prefix-{pvc uuid}": https://github.com/kubernetes-csi/external-provisioner/blob/master/pkg/controller/controller.go#L542:26